components/raftstore/src/store/worker/split_controller.rs

```diff
@@ -639,6 +858,49 @@ impl AutoSplitController {
             top_qps.push(qps);
         }
 
+        // Check if the top CPU usage region could be split.
+        // TODO: avoid unnecessary split by introducing the feedback mechanism from PD.
+        if !top_cpu_usage.is_empty() && !is_grpc_poll_busy {
```

Comment:
I think we also need to wait for a `detect_times` to avoid noise in here, it's may helpful for unnecessary split. cc @lhy1024